src/main/java/com/codingnomads/betty/logic/interfaces/FootballMatchesInfoJpaRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.codingnomads.betty.logic.interfaces;
+
+import com.codingnomads.betty.data.models.FootballMatchInfo;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.Date;
+import java.util.List;
+
+public interface FootballMatchesInfoJpaRepository extends JpaRepository<FootballMatchInfo, Long> {
+
+    @Query(value = "SELECT * FROM football_matches WHERE match_date >= NOW()", nativeQuery = true)
```

Comment:
This might be OK but you may get a SQL Exception or hang the database if you are using 'SELECT *'. It might be better to specify which fields you want instead. 'SELECT *' should be used mostly for debugging but not on production queries